shacl12-core/index.html

```diff
@@ -1422,25 +1422,24 @@ <h3>Property Shapes</h3>
-					<a>SHACL property path</a> or a <a>well-formed</a> <a>node expression</a>.</span>
-					<span data-syntax-rule="path-expr-order">The syntax rules for <a>SHACL property paths</a> take precedence over <a>node expressions</a>;
-					for example, this means that an <a>IRI</a> will be interpreted as a <a>predicate path</a> instead of an <a>IRI expression</a>.</span>
-					<span data-syntax-rule="path-not-literal">The <a>value</a> of <code>sh:path</code> cannot be a <a>literal</a>.</span>
+					<span data-syntax-rule="path-node">The <a>value</a> of <code>sh:path</code> in a property shape is a <a>well-formed</a>
+					<a>SHACL property path</a>.</span>
 					<br/><br/>
 					It is recommended, but not required, for a <a>property shape</a> to be declared as a <a>SHACL instance</a> of <code>sh:PropertyShape</code>.
 					<span data-syntax-rule="PropertyShape-path-minCount"><a>SHACL instances</a> of <code>sh:PropertyShape</code> have one <a>value</a> for the property <code>sh:path</code>.</span>
+					<br/><br/>
+					<span data-syntax-rule="path-values">A <a>property shape</a> has at most one <a>value</a> for the property <code>sh:values</code> and this <a>value</a> is a <a>well-formed</a> <a>node expression</a>.</span>
+					<span data-syntax-rule="path-defaultValue">A <a>property shape</a> has at most one <a>value</a> for the property <code>sh:defaultValue</code> and this <a>value</a> is a <a>well-formed</a> <a>node expression</a>.</span>
+					<span data-syntax-rule="path-values-iri">A <a>property shape</a> can only have a <a>value</a> for <code>sh:values</code> or <code>sh:defaultValue</code> when its <a>value</a> for <code>sh:path</code> is a <a href="#property-path-predicate">Predicate Path</a>.</span>
 				</p>
 				<p>
 					Informally, property shapes specify constraints that need to be met with respect to <a>nodes</a> that can be reached from the
-					<a>focus node</a> either by directly following a given property (specified as an <a>IRI</a>) or any other <a>SHACL property path</a>,
-					specified using <code>sh:path</code>.
+					<a>focus node</a> by either (a) directly following a given property (specified as an <a>IRI</a>), (b) directly following any other <a>SHACL property path</a>
+					specified using <code>sh:path</code>, or evaluating the node expression specified using <code>sh:values</code>, or (if no other values exist)
+					by evaluating the node expression specified using <code>sh:defaultValue</code>.
 				</p>
 				<p>
-					If the <code>sh:path</code> is not a <a>well-formed</a> property path, it must be a <a>node expression</a>
-					and the constraints will be validated against the <a>output nodes</a> of this node expression.
-					Note that support for node expressions is not required by SHACL Core,
+					Note that support for <code>sh:values</code> and <code>sh:defaultValue</code> is not required by SHACL Core,
```

Comment:
Can we just say:

```suggestion
                    Note that `sh:values` and `sh:defaultValue` are not used by SHACL Core,
SHACL Core,
```

Reply:
I prefer the pre-edit wording.  Presence in SHACL-Core notes the terms should be recognized as in the SHACL vocabulary, but knowing their functionality is explicitly "not required" is a clearer statement to me.

Reply:
Recognizing SHACL vocabulary" would imply some basic "support"?

I wanted to capture that they can be syntactically correct and not processed in the "SHACL Core profile" (to come...).